spec/models/piece_spec.rb

```diff
@@ -1,4 +1,49 @@
 require 'rails_helper'
 
 RSpec.describe Piece, type: :model do
+  describe '#obstructed?' do
+    it 'checks for diagonal obstruction between A6 and C4' do
+      game = FactoryGirl.create_obstructed_game
```

Comment:
Take a look at `let` statements. Will be useful for you here. 😄 

http://www.betterspecs.org/#let